libs/react-components/src/lib/components/Button/index.tsx

```diff
@@ -26,6 +26,8 @@ interface ButtonProps {
    * Click handler for the button
    */
   onClick?: () => void;
+
+  slot?: string;
```

Comment:
Can we add a description for the `slot` prop. You can copy it from the `IconButtonProrp` interface.